docs/hub/datasets-distilabel.md

```diff
@@ -0,0 +1,102 @@
+# Distilabel
+
+Distilabel is the framework for synthetic data and AI feedback for AI engineers that require high-quality outputs, full data ownership, and overall efficiency.
+
+Distilabel allows to build pipelines with any number of steps or tasks, that can be connected between, so the output of one step or task is fed as input to another. A series of steps that can be chained together to build complex data processing and generation pipelines with LLMs. The input of each step is a batch of data, containing a list of dictionaries, where each dictionary represents a row of the dataset, and the keys are the column names. To feeds data from and to the Hugging Face hub, we've defined a `Distiset` class as abstraction of a `datasets.DatasetDict`.
+
+## Distiset as special datasets.DatasetDict
+
+A Pipeline in distilabel returns a special type of Hugging Face `datasets.DatasetDict` which is called `Distiset`.
+
+The Distiset is a dictionary-like object that contains the different configurations generated by the Pipeline, where each configuration corresponds to each leaf step in the DAG built by the Pipeline. Each configuration corresponds to a different subset of the dataset. This is a concept taken from 🤗 datasets that lets you upload different configurations of the same dataset within the same repository and can contain different columns i.e. different configurations, which can be seamlessly pushed to the Hugging Face Hub.
+
+## Prerequisites
+
+First [login with your Hugging Face account](../huggingface_hub/quick-start#login):
+
+```bash
+huggingface-cli login
+```
+
+Make sure you have `distilabel` installed:
+
+```bash
+pip install -U distilabel[openai]
```

Comment:
Let's use `vLLM` or `TransformersLLM` instead as they are the only way to use Prometheus model with distilabel.